lib/oli_web/controllers/cognito_controller.ex

```diff
@@ -0,0 +1,158 @@
+defmodule OliWeb.CognitoController do
+  use OliWeb, :controller
+  import Oli.HTTP
+  import Oli.Utils
+  import Ecto.Changeset
+  require Logger
+
+  alias Oli.Accounts
+  alias Oli.Groups
+  alias Oli.Institutions
+  alias Oli.Institutions.SsoJwk
+
+  @jwks_relative_path "/.well-known/jwks.json"
+
+  def launch(conn, params) do
+    try do
```

Comment:
I like to be cautious here with this try/rescue. Do you know any example in which we can fall in this case? If so, it would be great to write a test and include the "unexpected error occurred" flow, wdyt?